ui/modules/reservation.ts

```diff
@@ -234,12 +234,14 @@ export const getReservationCancellationReason = (
 export const getNormalizedReservationOrderStatus = (
   reservation: ReservationType
 ): string | null => {
-  const orderStatuses = ["DRAFT", "PAID", "PAID_MANUALLY"];
-
   if (!reservation) return null;
 
-  if (orderStatuses.includes(reservation.orderStatus)) {
-    return reservation.orderStatus;
+  if (
+    !["CREATED", "WAITING_FOR_PAYMENT", "REQUIRES_HANDLING"].includes(
```

Comment:
Readability is kinda low, maybe create a helper function that is named appropriately. Negations infront of really long conditionals cause confusion.